src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -50,9 +59,16 @@ const CreateVMFooter: FC = () => {
   const [activeNamespace] = useActiveNamespace();
   const { instanceTypeVMState, setStartVM, setVM, startVM, vmNamespaceTarget } =
     useInstanceTypeVMStore();
-  const { selectedBootableVolume, selectedInstanceType, sshSecretCredentials, vmName } =
-    instanceTypeVMState;
+  const {
+    selectedBootableVolume,
+    selectedInstanceType,
+    sshSecretCredentials,
+    sysprepConfigMapData,
+    vmName,
+  } = instanceTypeVMState;
   const { applyKeyToProject, secretOption, sshPubKey, sshSecretName } = sshSecretCredentials || {};
+  const defaultPreferenceName = getLabel(selectedBootableVolume, DEFAULT_PREFERENCE_LABEL);
+  const isWindowsOSVolume = defaultPreferenceName?.startsWith(OS_WINDOWS_PREFIX) || false;
```

Comment:
no need for || false

Reply:
btw didnt we have a way to check the label on the instancetype preference to know if it a windows VM?

Reply:
Oh, `false` should not be there for sure, thanks!

> btw didnt we have a way to check the label on the instancetype preference to know if it a windows VM?

I think I found something (not remember exactly) but it wasn't working the way we need here. Hence I did here what I did. Maybe I should better move this check to a separate function/hook?

Reply:
So we can use `useIsWindowsBootableVolume` here in this file :)